README.md

```diff
@@ -14,7 +14,7 @@ Currently available only for scala 2.11. In your build.sbt file,
 ```sbt
 resolvers += Resolver.jcenterRepo
 
-libraryDependencies += "io.ticofab" % "aws-request-signer_2.11" % "0.4.1"
+libraryDependencies += "io.ticofab" %% "aws-request-signer" % "0.4.1"
```

Comment:
You most likely wouldn't be able to. That is exactly why I want to make this visible in the README. I opened one issue about moving to 2.12 :)